### PR TITLE
organization details screen base

### PR DESCRIPTION
**Pull request by daileytj**
Fixes BLUI-4849

#### Changes proposed in this Pull Request:

- Add organization details screen base

#### Screenshots / Screen Recording (if applicable)

Screenshot 2023-11-22 at 9 05 51 AM

#### To Test:

- yarn start:example 
- navigate to localhost:3000/organization-details-base-screen-demo

### CODE REVIEW

File: login-workflow/example/src/screens/demo-components/OrganizationDetailsScreenBaseDemo.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        name: 'Afghanistan',
+        isoCode: 'AF',
+        flag: '🇦🇫',
+        phonecode: '93',
+        currency: 'AFN',
+    },
+    {
+        name: 'Aland Islands',
+        isoCode: 'AX',
+        flag: '🇦🇽',
```
Comment (ektaghag-eaton) [inline login-workflow/example/src/screens/demo-components/OrganizationDetailsScreenBaseDemo.tsx:26]: Didn't see the use of this other isoCode, flag, phonecode and currency. Are we going to use this on another screen?

**Comment by ektaghag-eaton** [comment]
1. No states are under `American Samoa` and `Aland Islands` 
Screenshot 2023-11-23 at 10 44 18 AM

2. When I select `Aland Islands`, it shows an error in the `State` dropdown. After this, if I select `Afghanistan` still shows an error in the state dropdown (even if it has a list of states). I think we Need to reset the error state of the `State` dropdown when the country changes.

File: login-workflow/example/src/screens/demo-components/OrganizationDetailsScreenBaseDemo.tsx (hunk 1 above)
Comment (daileytj) [reply login-workflow/example/src/screens/demo-components/OrganizationDetailsScreenBaseDemo.tsx:26]: These constants were provided from the DERMS project. I simply copied them so there may be issues with the data set or unused keys

**Comment by JeffGreiner-eaton** [comment]
I get what we are trying here but when we change country value the state is no longer valid and we throw warning...

image

Would the freeSolo prop for autocomplete be of any help here?
https://mui.com/material-ui/react-autocomplete/#free-solo

**Comment by daileytj** [comment]
> I get what we are trying here but when we change country value the state is no longer valid and we throw warning...
> 
> image
> 
> Would the freeSolo prop for autocomplete be of any help here? https://mui.com/material-ui/react-autocomplete/#free-solo

This does take away the warning, but causes some other issues. One, they can enter any value they want so that may not be ideal for validation. There also becomes a type issue as the standard value for state is and object and the freesolo event only outputs a string value. We may be able to work around that as well. I'll look into what we can do here a bit more.

**Review by JeffGreiner-eaton — APPROVED** [review]
looks good to me

**Comment by daileytj** [comment]
@ektaghag-eaton this is ready for your review

**Review by ektaghag-eaton — APPROVED** [review]
Working fine 👍

**Pull request merged**